app/src/main/java/de/westnordost/streetcomplete/MainActivity.java

```diff
@@ -494,6 +494,10 @@ else if(e instanceof OsmAuthorizationException)
 
 		@AnyThread @Override public void onFinished()
 		{
+			if (Prefs.Autosync.valueOf(prefs.getString(Prefs.AUTOSYNC,"ON")) == Prefs.Autosync.OFF)
+			{
+				Toast.makeText(MainActivity.this, "Finished uploading changes", Toast.LENGTH_SHORT).show();
```

Comment:
Please don't hardcode strings! Otherwise they can't be translated...
see https://developer.android.com/guide/topics/resources/localization.html for more information